tests/test_base.cpp

```diff
@@ -562,7 +580,7 @@ std::string server_node::config_path() const
 	return m_path;
 }
 
-server_config server_node::config() const
+server_config &server_node::config()
```

Comment:
It is needed to modify server_config. Modifying is needed to make backend serializable and update config file.